docs/pages/tabs.md

```diff
@@ -146,13 +146,16 @@ Add the attribute `data-active-collapse="true"` to a tabstrip to collapse active
 
 ### Browser history
 
-The current state of the tabset is now recorded by adding a hash with the tab panel ID to the browser URL when a tab is opened. By default, tabs replace the browser history. Modify this behavior by using attribute `data-update-history="true"` to *append* to the browser history. In the latter case the browser back button will track each click that opens a tab panel.
+When the `data-deep-link="true"` attribute is set, the current state of the tabset is recorded by adding a hash with the tab panel ID to the browser URL when a tab opens. By default, tabs *update and replace* the browser history (meaning that the browser does not navigate). Modify this behavior by using attribute `data-update-history="true"` to *append* to the browser history. In the latter case the browser back button will track each click that opens a tab panel.
```

Comment:
I guess "When the `data-deep-link` attribute is set to `true`" is most correct.

I'm not sure I really understand the technical distinction between 'history.updateState()' and 'history.replaceState()' either, which is what I was trying to talk around. It's probably best just to get rid of the parenthetical or link to the Mozilla documentation.